packages/driver/src/cy/ensures.ts

```diff
@@ -395,7 +395,7 @@ export const create = (state: StateFunc, expect: $Cy['expect']) => {
     if (!isRunnerAbleToCommunicateWithAut()) {
       const crossOriginCommandError = $errUtils.errByPath('miscellaneous.cross_origin_command', {
         commandOrigin: window.location.origin,
-        autOrigin: state('autLocation').originPolicy,
+        autOrigin: state('autLocation').superDomainOrigin,
```

Comment:
```suggestion
        autSuperDomainOrigin: state('autLocation').superDomainOrigin,
```

Reply:
should be fixed in 1ee8e686a41d0bb4e723c6faf4df4a301636edea